### PR TITLE
tiup: add 3 docs dm-deploy/destroy/audit

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
add 3 TiUP docs:
- tiup/tiup-component-dm-audit.md
- tiup/tiup-component-dm-destroy.md
- tiup/tiup-component-dm-deploy.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4868
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Joyinqin** [comment]
Wait for the translation of the `tiup-dm-topology-reference.md` to fix the dead link.

File: tiup/tiup-component-dm-deploy.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+The `tiup dm deploy` command is used to deploy a new cluster.
+
+## Syntax
+
+```shell
+tiup dm deploy <cluster-name> <version> <topology.yaml> [flags]
+```
+
+- `<cluster-name>`: the name of the new cluster, which cannot be the same as the existing cluster names.
+- `<version>`: the version number of the TiDB cluster to deploy, such as `v2.0.0`.
```
Comment (ran-huang) [inline tiup/tiup-component-dm-deploy.md:16]:
```suggestion
- `<version>`: the version number of the DM cluster to be deployed, such as `v2.0.0`.
```

File: tiup/tiup-component-dm-deploy.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- `<topology.yaml>`: the prepared [topology file](/tiup/tiup-dm-topology-reference.md)
+
+## Options
+
+### -u, --user
+
+- Specifies the user name used to connect to the target machine. This user must have the secret-free sudo root permission on the target machine.
```
Comment (ran-huang) [inline tiup/tiup-component-dm-deploy.md:23]:
```suggestion
- Specifies the user name used to connect to the target machine. This user must have the password-free sudo root permission on the target machine.
```

File: tiup/tiup-component-dm-deploy.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+### -i, --identity_file
+
+- Specifies the key file used to connect to the target machine.
+- Data type: `STRING`
+- Default: `~/.ssh/id_rsa`
+
+### -p, --password
+
+- Specifies the password used to connect to the target machine. Do not use it with `-i/--identity_file` at the same time.
```
Comment (ran-huang) [inline tiup/tiup-component-dm-deploy.md:35]:
```suggestion
- Specifies the password used to connect to the target machine. Do not use this option and `-i/--identity_file` at the same time.
```

File: tiup/tiup-component-dm-destroy.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+After an application goes offline, if you want to release the machines occupied by the cluster for use by other applications, you need to clean up the data on the cluster and the deployed binary files. To destroy the cluster, the `tiup dm destroy` command performs the following operations:
+
+- Stops the cluster.
+- For each service, delete its log directory, deployment directory, and data directory.
+- If the parent directory of the data directory or deployment directory of each service is created by `tiup-dm`, also delete the parent directory.
+
+## Syntax
+
+```sh
```
Comment (ran-huang) [inline tiup/tiup-component-dm-destroy.md:15]:
```suggestion
```shell
```

File: tiup/tiup-component-dm-destroy.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+tiup dm destroy <cluster-name> [flags]
+```
+
+`<cluster-name>`: the name of the cluster to destroy.
```
Comment (ran-huang) [inline tiup/tiup-component-dm-destroy.md:19]:
```suggestion
`<cluster-name>`: the name of the cluster to be destroyed.
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

File: tiup/tiup-component-dm-deploy.md (hunk 2 above)
Comment (Joyinqin) [reply tiup/tiup-component-dm-deploy.md:23]: both are OK~

**Review by ran-huang — APPROVED** [review]

**Pull request merged**